### ISSUE TITLE
Possibility to animate individual cards

### ISSUE THREAD

**Issue by Bort89** [j-a-n/lovelace-wallpanel#52]
First things first: this module is awesome! I am recreating my Dakboard and making great progress! :)

I have a HA dashboard on a 24 inch screen in the center of my livingroom but most of the time it shows this excellent wallpanel with pictures from Google photos. It also shows a calendar, time, weather, etc. etc.

You can imagine that I want the calendar to stay in place while other cards may animate their position.
Is this possible? I tried with css to the wp_style and with your options to the same wp_style but didn't get it to work.

TLDR; Is it possible to animate individual cards?

Keep up the good work!

**Comment by j-a-n** [comment]
With version 4.4 I added the ability to set the parent element of each card by element id.
For example:
```yaml
  cards:
    - type: markdown
      content: Fixed position
      wp_style:
        parent: wallpanel-screensaver-container
        position: fixed
        top: 10px
        left: 10px
        width: 300px
        height: 200px
```
You can play around with it and give me feedback.

**Comment by Bort89** [comment]
Thanks. I added the part and it seems like the position is indeed fixed but the card still moves around the screen like the rest. Only now the begin position is set with position.
Is it supposed to stop moving when it's set to a parent?

**Comment by j-a-n** [comment]
Yes, the card should stay at it's initial position.
What does your `wp_style` config looks like?

**Comment by Bort89** [comment]
wp_style:
         width: 2100px
        grid-row: 5
        grid-column: 1 / span 5
        margin-top: '-50px'
        margin-left: '-50px'
        padding-top: 20px
        padding-right: 400px
        padding-left: 60px
        background-image: linear-gradient(to top, rgba(0,0,0,0.75) 60%, rgba(0,0,0,0) 100%)
        font-size: 20px
        font-weight: 600
        '--primary-text-color': '#e1e1e1'
        border-radius: 10px
        parent: wallpanel-screensaver-container
        position: fixed
        top: 250px
        left: 10px

Sorry, I don't know how to format it here.

**Comment by j-a-n** [comment]
Is the latest version installed?
Perhaps a caching issue?

**Comment by Bort89** [comment]
Correct my good sir! It was indeed a caching issue, which is weird because I did ctrl+f5 several times.
Anyway: it does indeed work butttt I lose (or get) a new style in the cards. See the image attached where you can see the two situations: one before the parent and one after. Probably because the parent has a different style sheet?
![Image 2640](https://user-images.githubusercontent.com/31632020/203634627-6a885524-32b9-434e-ba93-11f3c7d0fa02.png)

**Comment by j-a-n** [comment]
Okay, please try the new version 4.5.
Please start with this wp_style configuration for the fixed card:
```yaml
 wp_style:
   parent: wallpanel-screensaver-fixed-info-box-content
```

**Comment by Bort89** [comment]
That's purdy my friend! Works like a charm! :)
